### PR TITLE
[Process] Add getter for process starttime

### PR DESCRIPTION
**Pull request by dompie**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | issue #35531
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Add a getter for process start time.

### CODE REVIEW

File: src/Symfony/Component/Process/Process.php [hunk 1]
```diff
@@ -1621,4 +1621,9 @@ private function getDefaultEnv(): array
 
         return $env;
     }
+
+    public function getStartTime(): ?float
```
Comment (ro0NL) [inline src/Symfony/Component/Process/Process.php]: i tend to prefer a non-nullable API, thus a throw if `!isStarted()`.

Comment (nicolas-grekas) [reply src/Symfony/Component/Process/Process.php]: yet it makes sense to return null when the process didn't start

Comment (nicolas-grekas) [reply src/Symfony/Component/Process/Process.php]: (but I'm fine with throwing a LogicException as some other methods do so)
Please group the method with other public ones, before private and protected.

Comment (dompie) [reply src/Symfony/Component/Process/Process.php]: > i tend to prefer a non-nullable API, thus a throw if `!isStarted()`.
I tend to prefer nullable API - it allows greater range of use and is more comfortable IMHO.

Comment (ro0NL) [reply src/Symfony/Component/Process/Process.php]: the issue is its still a nullable , even if the process is started. Thats the common case IMHO ... in this case you dont want to account for null all the time.

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
(oh, can you please add a line in the changelog of the component?)

**Comment by dompie** [comment]
:heavy_check_mark: add changelog note

**Comment by nicolas-grekas** [comment]
Thank you @dompie.

**Pull request merged**